src/elm/Message/LabeledMsg.elm

```diff
@@ -0,0 +1,40 @@
+module Message.LabeledMsg exposing (decoder, encode)
```

Comment:
This module provides basic support for a standard message format between Elm & JS/TS:

```javascript
{
  msgType: "MessageType",
  msg: { ... message data here ...}
}
```